sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Rules/Hotspots/ControllingPermissionsBase.cs

```diff
@@ -68,7 +72,7 @@ protected override void Initialize(SonarAnalysisContext context)
                     KnownType.System_Security_Permissions_PrincipalPermissionAttribute));
 
             BaseTypeTracker.Track(context,
-                BaseTypeTracker.WhenDerivesFrom(
+                BaseTypeTracker.MatchSubclassesOf(
```

Comment:
I am a bit confused by the naming. The previous method has been reworded to use `WhenDerived...` and this is to `MatchSubclasses`. I'd prefer having the same kind of name in all contexts.